go.mod

```diff
@@ -44,7 +56,7 @@ require (
 	k8s.io/utils v0.0.0-20201110183641-67b214c5f920
 	knative.dev/hack v0.0.0-20211102174540-b284d49386cc
 	knative.dev/hack/schema v0.0.0-20211102174540-b284d49386cc
-	knative.dev/pkg v0.0.0-20211103135647-464034912f7e
+	knative.dev/pkg v0.0.0-20211101212339-96c0204a70dc
```

Comment:
using an older version?

Reply:
For some reason it downgrade it also when running go get ginkgo 1.14.2